README.md

```diff
@@ -1,26 +1,190 @@
-# Service Fabric CLI (sfctl)
+# Service Fabric Cross Platform Command-line Interface
 
-Command-line interface for interacting with Azure Service Fabric clusters and
-their related entities.
+Command-line interface(CLI) for interacting with Azure Service Fabric clusters and their related entities.
 
-## Installation
+## Compatibility between sfctl and Service Fabric versions
+sfctl is forward compatible with newer versions of Service Fabric runtime i.e. an older version of sfctl can work with newer versions of Service Fabric runtime but will not support new commands available in later versions. This also means that newer versions of sfctl will not be able to communicate with older versions of Service Fabric.
 
-```bash
-pip install sfctl
+Following table gives the mapping of sfctl versions and Service Fabric runtime versions.
+<table>
+<thead>
+<tr>
+<th>sfctl version</th><th>Service Fabric runtime version</th>
+</tr>
+</thead>
+<tr><td>1.1.0</td><td>5.6.130 or later</td></tr>
+<tr><td>2.0.0</td><td>6.0 and later</td></tr>
+</table>
+	
+## Installing sfctl using pip

```

Comment:
If we want to add more documentation, let's do it in the Azure Docs repository, not in this readme. That way we cut down on the amount of duplication. My recommendation would be to link back to this page: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cli